lib/VeresOne.js

```diff
@@ -113,6 +113,24 @@ class VeresOne {
     return operation;
   }
 
+  /**
+   * Sets the id of a given key. Used by `did-io` drivers.
+   *
+   * @param {LDKeyPair} key
+   * @param {string} [did] - Optional DID.
+   * @param {string} [didType] - 'nym' or 'uuid'
+   * @param {string} [mode] - 'test', 'dev', 'live' etc.
+   *
+   * @returns {string} Returns the key's id.
+   */
+  setKeyId({key, did, didType = constants.DEFAULT_DID_TYPE, mode = this.mode}) {
```

Comment:
I'm not a huge fan of this name -- because set it to what? I don't think someone looking at this API would know what it does. And if you don't pass in a `did` it still gets set somehow (to what?)? Feels like magic. Maybe this is more like a "computeKeyId"/"setKeyIdToSomething"/"useSomethingForKeyId"/"includeDidInKeyId"/"updateKeyId" (the latter because you're supposed to call it after creating a DID or something)?

Something just feels off and suboptimal here.

Reply:
Makes sense, agreed. Pondering..

Reply:
Renamed.